app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -415,6 +401,13 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
         createRepository()
     }
 
+    private fun runGitOperation(operation: Int) = lifecycleScope.launch {
+        launchGitOperation(operation).fold(
+            onSuccess = { refreshPasswordList() },
```

Comment:
This now also refreshes during push where its not strictly needed but it simplified the code so I let it be.

Reply:
Totally fine, refreshing is reasonably cheap.